public class tryCatch {
    public static void main(String[] args) {
        try {
            float x = 5;
            System.out.println(x/2);
        } catch (Exception e) {
            // TODO: handle exception
            System.out.println("Error was found");
        }
    }
}
